Documentation/schemas/profile-manifest-v0.md

```diff
@@ -24,11 +25,13 @@ It contains an ordered list of images (name + reference) to a be applied on a sy
 - value/images/#/image: string, compatible with OCI image name specs.
   Name of the image to unpack.
 - value/images/#/reference: string, compatible with OCI image reference specs.
-  Referenced image must be available in the storepath, as a file name `${image}:${reference}.torcx.tgz`.
+  Referenced image must be available in the storepath, as a file name `${image}:${reference}.torcx.${format}`
+- value/images/#/format : string, either "tgz" or "squashfs" to indicate the image is stored as either a gzipped tarball or squashfs archive.
+  For backwards compatibility reasons, format will default to "tgz", but "squashfs" should be preferred for new images.
```

Comment:
`squashfs` supports had a few (deprecated) major versions and it also supports multiple compression versions, which we should probably specifiy here. I'm mostly concerned with the latter.

Would you prefer to pick a specific compression algorithm, or to have multiple formats with compression identifiers (i.e. `sqsh.gz`)? I'd lean toward the former.

Reply:
I also lean towards the former. I can rebase on #105 and indicate the single supported version+compression algorithm.